includes/blocks/class-sensei-block-quiz-question.php

```diff
@@ -22,21 +22,53 @@ public function __construct() {
 		Sensei_Blocks::register_sensei_block(
 			'sensei-lms/quiz-question',
 			[
-				'render_callback' => [ $this, 'render_quiz_question' ],
+				'render_callback' => [ $this, 'render_empty_string' ],
```

Comment:
Just a thought, do you think it's possible to use the `__return_empty_string` WP function here instead of creating a new one?

Reply:
Good catch, thanks @Imran92!
Updated here: https://github.com/Automattic/sensei/pull/7643/commits/65b05b63534e291b4cd190ebd5da47e8a3178f99